src/components/ProgressIndicator/utils.js

```diff
@@ -57,24 +57,25 @@ const addFixedSteps = (
   const summaryStep = {
     to: 'overzicht',
     label: intl.formatMessage(STEP_LABELS.overview),
-    isCompleted: isConfirmation,
+    isCompleted: isPayment,
```

Comment:
For the case where no payment is required, we go straight to the confirmation view (which doesn't show the progress indicator)